### ISSUE TITLE
Flags as arguments

### ISSUE THREAD

**Issue by gfilipekint** [NervanaSystems/ngraph-tf#411]
Hi.
I think it would be a good idea to enable users to change which flags are on during the build process using `build_ngtf.py` script. Currently it is quite troublesome to edit script manually just to enable IntelGPU or GPU. Especially if you want to automate building process (using docker etc) it is inconvenient that you have to `sed` the file just to change one flag.
I understand that this is of low priority, but it's definitely something that would make ngraph+tf more user-friendly.

**Comment by avijit-nervana** [comment]
Yes - we will definitely consider this. What flags do you want to be parametrizable?

**Comment by gfilipekint** [comment]
For example enabling GPU would be convenient as a flag, maybe enabling unit tests or debug version etc. Basically the most common flags an user would maybe want to change, it's up to you to decide, I just wanted to bring this to your attention , but to be honest this issue is not of great significance.
Best regards :)
